src/test/codegen/debug-column-1-based.rs

```diff
@@ -0,0 +1,18 @@
+// ignore-windows
+
+// compile-flags: -g -C no-prepopulate-passes
+
+// CHECK-LABEL: DISubprogram(name: "main"
+// CHECK:       {{.*}}DILocalVariable(name: "x",{{.*}}line: 15{{.*}}
+// CHECK-NEXT:  {{.*}}DILexicalBlock{{.*}}
```

Comment:
Maybe turn this line into a test for the `DILexicalBlock` change?

Reply:
I'm not that fast okay :smile:?

Reply:
I was caught between commenting too early and risking that you already want to do it or commenting too late at which point you might already be doing something else. :).